sympy/polys/polytools.py

```diff
@@ -71,6 +71,9 @@ def wrapper(f, g):
         g = _sympify(g)
         if isinstance(g, Poly):
             return func(f, g)
+        elif isinstance(g, Integer):
```

Comment:
I assume this is where you special cased Integer (as you referred to in how you fixed the previously commented out tests). Can `g` be an int or is that always sympified before it gets here?

Reply:
`g` is sympified 3 lines up from here.